boringssl-static/pom.xml

```diff
@@ -75,6 +75,282 @@
   </dependencies>
 
   <profiles>
+
+    <!-- The profile that builds a fips-boringssl-static jar -->
+    <profile>
+      <id>fips-boringssl-static</id>
+      <properties>
+        <boringsslCheckoutDir>${project.build.directory}/boringssl-${boringsslBranch}/boringssl</boringsslCheckoutDir>
+        <boringsslBuildDir>${boringsslCheckoutDir}/build</boringsslBuildDir>
+        <!--   Latest FIPS compliant boringSSL commit   -->
+        <boringsslBranch>${env.BORINGSSL_COMMIT}</boringsslBranch>
```

Comment:
Why not just "hardcode" it the same as we do with the other boringssl build ?

Reply:
@normanmaurer In sometime BooringSSL might release another FIPS validated build. That is why i feel we should not hardcode it. https://boringssl.googlesource.com/boringssl/+/master/crypto/fipsmodule/FIPS.md

Reply:
@normanmaurer Thanks. Updated commit